components/web/apps/org.wso2.carbon.apimgt.publisher/src/main/component.yaml

```diff
@@ -1,9 +1,9 @@
-apis: []
-
-
-config:
-  title : "API Publisher Application"
-  appName : API Publisher
-  loginRedirectUri: "/apis"
-  loginPageUri : "https://localhost:9292/publisher/auth/login"
-
+apis: []
+
+

```

Comment:
Thanks @rmsamitha for the suggestion , I have removed it with this [commit](https://github.com/wso2/carbon-apimgt/pull/4050/commits/4e0e1dfea11ce0c762e30031a6ddba4e4681c120)